pomchecker-cli/src/main/java/org/kordamp/maven/checker/cli/internal/PomParser.java

```diff
@@ -85,8 +88,21 @@ private static MavenProject createMavenProject(File pomFile, Context context, Pl
                 mavenExecutionRequest.getProjectBuildingRequest();
 
             projectBuildingRequest.setRepositorySession(context.repositorySystemSession());
-            projectBuildingRequest.setRemoteRepositories(context.remoteRepositories()
-                    .stream().map(PomParser::toArtifactRepository).collect(Collectors.toList()));
+            List<ArtifactRepository> remoteRepositories = context.remoteRepositories()
+                    .stream().map(PomParser::toArtifactRepository).collect(Collectors.toList());
+
+            // Add deployment repository to resolve the artifacts in the project
+            MavenArtifactRepository thisRepository = new MavenArtifactRepository();
+            thisRepository.setId("deployment");
+            thisRepository.setUrl(context.basedir().toUri().toString());
```

Comment:
What is the location to where this `basedir` property points to?

Reply:
It's set as the directory where the command is run, meaning that it will not work if you execute from a different directory

Reply:
Got it. So if JReleaser invokes Pomchecker at the root of the staging directory things will work out as expected.

Reply:
However, that's not the case looking at https://github.com/jreleaser/jreleaser/blob/20948c90a32ac50f105c6065785877d9ba971a53/sdks/jreleaser-java-sdk-commons/src/main/java/org/jreleaser/sdk/commons/AbstractMavenDeployer.java#L252-L256

One [overcomplicated] way to test the fix in production would be:
- build your own pomchecker version
- install the CLI locally (may use sdkman for this)
- configure the new pomchecker version in jreleaser.yml
- attempt a deployment in dry run mode

Reply:
I tried building from my fork (https://github.com/gastaldi/pomchecker) but it fails because the tool tries to download from https://github.com/kordamp/pomchecker/releases/download/v999/pomchecker-999.zip (see https://github.com/quarkiverse-release/quarkiverse-release/actions/runs/10634046170/job/29480500201)

Reply:
@aalmiray you're saying that if I have something like: 

```bash
.
├── jreleaser.yml
└── repository
    └── io
        └── quarkiverse
            └── slack
                ├── **quarkus-slack**
                ...
```		

The basedir will be the root directory and not `repository`? If so, that would not work.